lib/uplink/example_test.go

```diff
@@ -0,0 +1,286 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"storj.io/storj/lib/uplink"
+	"storj.io/storj/pkg/storj"
+)
+
+const (
+	myAPIKey = "change-me-to-the-api-key-created-in-satellite-gui"
+
+	satellite       = "mars.tardigrade.io:7777"
+	myBucket        = "my-first-bucket"
+	myUploadPath    = "foo/bar/baz"
+	myData          = "one fish two fish red fish blue fish"
+	myEncryptionKey = "you'll never guess this"
+)
+
+// ExampleProj_CreateBucket example documentation
+func ExampleProject_CreateBucket() {
+	var encryptionKey storj.Key
+	copy(encryptionKey[:], []byte(myEncryptionKey))
+
+	apiKey, err := uplink.ParseAPIKey(myAPIKey)
+	if err != nil {
+		log.Fatalln("could not parse api key:", err)
+	}
+
+	ctx := context.Background()
+
+	// Create an Uplink object with a default config
+	upl, err := uplink.NewUplink(ctx, nil)
+	if err != nil {
+		fmt.Println("could not create new Uplink object:", err)
+	}
+	defer func() {
+		if err := upl.Close(); err != nil {
+			log.Printf("Failed to close input: %scope", err)
+		}
+	}()
+
+	// It is temporarily required to set the encryption key in project options.
+	// This requirement will be removed in the future.
+	opts := uplink.ProjectOptions{}
+	opts.Volatile.EncryptionKey = &encryptionKey
+
+	// Open up the Project we will be working with
+	proj, err := upl.OpenProject(ctx, satellite, apiKey, &opts)
+	if err != nil {
+		fmt.Println("could not open project:", err)
```

Comment:
same comment for all error handling in all of these examples. this is bad because the program will continue

Reply:
changed to use log.Fatal()